index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <!-- Google Icons --> 
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="root">
+        <header class="top_header">
+            <nav class="nav_bar">
+                <div class="left_side">
+                    <a href="#" class="logo">H.Monda</a>
+                </div>
+                <div class="right_side">
+                    <i class="material-icons menu_icon">menu</i>
+                </div>
```

Comment:
- [ ] Please nest the #hamburger menu# button withing <nav> tag not a <div> tag. This allows screen reader users to navigate directly to the navigation landmark